### DISCUSSION TITLE
I can see the translate button, but it does not work, and show a error in Russian , how to fix it ?

### DISCUSSION THREAD

**Discussion by Andrew2Neo** [ilyhalight/voice-over-translation#422] [Q&A]
<img width="879" alt="iShot_2023-11-20_23 53 38" src="https://github.com/ilyhalight/voice-over-translation/assets/127911537/7dc658da-d1a1-4392-9b0c-58e6c354d802">

**Comment by NullVerdict** [comment]
Install beta

**Comment by YuchkinUA** [comment]
@ilyhalight hi! 
I have a problem, the plugin throws an error when translating.
What happened?
and what to do with it?

thanks.)
![232](https://github.com/ilyhalight/voice-over-translation/assets/127149244/fcbd912e-cae8-4601-bca6-000c1b8492b9)
![2332](https://github.com/ilyhalight/voice-over-translation/assets/127149244/c94542a5-040f-4c34-963b-b4fec7cc78de)

**Comment by NullVerdict** [comment]
> @ilyhalight hi! I have a problem, the plugin throws an error when translating. What happened? and what to do with it?
> 
> thanks.) ![232](https://private-user-images.githubusercontent.com/127149244/285992420-fcbd912e-cae8-4601-bca6-000c1b8492b9.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDExMTU3OTQsIm5iZiI6MTcwMTExNTQ5NCwicGF0aCI6Ii8xMjcxNDkyNDQvMjg1OTkyNDIwLWZjYmQ5MTJlLWNhZTgtNDYwMS1iY2E2LTAwMGMxYjg0OTJiOS5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMTI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTEyN1QyMDA0NTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04OWM3MzU0MDlkMjRmN2NlZTFhMmVkMzJhYzM1MzEyMjFlOTNmZThkMWU2ZDY4YjZlYzczNDE1Y2ZmNjhkMmFkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.fwHPmElRRT9_6nXm52ZtF32otjhiPD6d4Sn0jnNhIhY) ![2332](https://private-user-images.githubusercontent.com/127149244/285992430-c94542a5-040f-4c34-963b-b4fec7cc78de.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDExMTU3OTQsIm5iZiI6MTcwMTExNTQ5NCwicGF0aCI6Ii8xMjcxNDkyNDQvMjg1OTkyNDMwLWM5NDU0MmE1LTA0MGYtNGMzNC05NjNiLWI0ZmVjN2NjNzhkZS5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMTI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTEyN1QyMDA0NTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMTUyNGIyMWJmMjc2YjIwMzU3ZTQwYWNiYjg2NjZiN2UwMDE1ZTk3OTJkMDNiMGM5NzM2MWQzMjBmMWEyMjcwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.WGq9KE_xKWexC4ALsGZeklfIk7JHcr8Ks0YqeHT2FOU)

Upgrade to beta and enable audio proxying (if not enabled) Also, perhaps the problem is that Yandex has disabled support for many languages (including Ukrainian) in the voice translation.

**Comment by YuchkinUA** [comment]
@ilyhalight @SashaXser Oh, sorry.
Didn't see the post...
As I understand it, Ukrainian has stopped working from the list of languages.

Well, sad :(
I will wait for the next updates...

PS But the plugin still works great! :)

![image](https://github.com/ilyhalight/voice-over-translation/assets/127149244/8b9a4f32-3bb0-43d5-93e0-dbf4aabb083f)

![image](https://github.com/ilyhalight/voice-over-translation/assets/127149244/5c46dd1f-b7ae-4db2-8afd-83bb4d1ef99e)
